theoplayer/getting-started/01-sdks/03-ios/00-getting-started.mdx

```diff
@@ -0,0 +1,196 @@
+# Getting started on iOS/tvOS
+
+## Introduction
+
+This guide covers the steps to obtain and setup the core THEOplayer SDK targeting a native iOS/tvOS application. It also provides information and usage of the commonly used APIs.
+
+## Setup
+
+### THEOplayer license
+
+THEOplayer requires a valid THEOplayer license to be configured in order to access the SDK APIs (with the exception of some whitelisted sources hosted by our CDN). Request your license via [https://portal.theoplayer.com](https://portal.theoplayer.com).
+
+After creating your license, you can copy its license string to your clipboard as demonstrated in the screenshot below.
+
+![](https://cdn.theoplayer.com/images/git/theoplayer-ios-sdk-license-string.png)
+
+> **Note:** It's possible to add multiple platforms in a single license ie. iOS & tvOS.
+
+Later in the guide we explain how to configure the license ([in this section](#adding-the-theoplayer-license)).
+
+### Installation
+
+The THEOplayer SDK is published on the following package managers:
+
+#### <span style={{ fontSize: '1.25rem' }}>Cocoapods</span>
```

Comment:
Why the odd font size?

Reply:
`####` which transforms to an `h4` tag has a smaller `font-size` than regular text, which is strange for headings that are designed to headline a section.

Reply:
Maybe we should fix that more globally then in [src/css/custom.css](https://github.com/THEOplayer/documentation/blob/main/src/css/custom.css).

Reply:
I thought `--ifm-h3-font-size` is `1.5rem` ?
<img width="585" alt="h3" src="https://github.com/user-attachments/assets/ecdc5c49-6636-4418-b4d0-74d15aade791" />